site/content/en/docs/concepts/admission_check.md

```diff
@@ -128,3 +128,4 @@ If any of the Workload's AdmissionCheck is in the `Rejected` state:
 ## What's next?
 
 - Read the [API reference](/docs/reference/kueue.v1beta1/#kueue-x-k8s-io-v1beta1-AdmissionCheck) for `AdmissionCheck`
+- Learn how to use `adminssionCheck` on [Provisioning Admission Check Controller](/docs/admission-check-controllers/provisioning/#setup)
```

Comment:
```suggestion
- Learn how to use `admissionCheck` on [Provisioning Admission Check Controller](/docs/admission-check-controllers/provisioning/#setup)
```
But maybe we can say:
```
Learn more from the built-in [Provisioning Admission Check Controller](/docs/admission-check-controllers/provisioning)
```
WDYT?

Reply:
Thank you for your suggestion! It looks more concise.